.circleci/config.yml

```diff
@@ -204,7 +252,35 @@ jobs:
           name: Push ${CIRCLE_TAG} tag
           command: |
               docker push quay.io/3scale/apicast-operator:${CIRCLE_TAG}
-  run-operator-e2e-test:
+  run-k8s-e2e-test:
+    environment: CHANGE_MINIKUBE_NONE_USER=true
+    machine:
+      image: ubuntu-1604:201903-01
+    steps:
+      - kubernetes/install-kubectl
+      - start-minikube
+      - install-golang
+      - install-operator-sdk
+      - checkout
+      - install-operator-dependencies
+      - attach-workspace
+      - run:
+          name: Unpack and use local images by re-using the Docker daemon
+          command: |
+            docker load -i /tmp/workspace/images/operator-image.tar
+      - kubernetes/create-or-update-resource:
+          resource-file-path: test/resources/namespace-operator-test.yaml
+          show-kubectl-command: true
+      - watch-operator-pod
+      - watch-operator-pod-logs
+      - run:
+          name: run E2E tests
+          command: |
+            make e2e-run IMAGE=172.30.1.1:5000/openshift/apicast-operator:${CIRCLE_SHA1} NAMESPACE=operator-test
```

Comment:
In other places of the file I've seen references to 172.30.1.1:5000/openshift/apicast-operator:test instead of ${CIRCLE_SHA1}. Specifically in  tag-operator-image-master and tag-operator-image-release jobs. Shouldn't be all :test or all :{CIRCLE_SHA1} ?

Reply:
:+1: